src/mobu/business/notebookrunner.py

```diff
@@ -61,73 +60,94 @@ def clone_repo(self) -> None:
         with self.timings.start("clone_repo"):
             self._repo = git.Repo.clone_from(url, path, branch=branch)
 
+    def find_notebooks(self) -> List[Path]:
+        with self.timings.start("find_notebooks"):
+            notebooks = [
+                p
+                for p in Path(self._repo_dir.name).iterdir()
+                if p.suffix == ".ipynb"
+            ]
+            if not notebooks:
+                msg = "No notebooks found in {self._repo_dir.name}"
+                raise NotebookRepositoryError(msg)
+            SystemRandom().shuffle(notebooks)
```

Comment:
I think we might want to make the shuffle configurable.  I can imagine some sort of use-case where we wanted to ensure the order in which we were running things.

Reply:
I agree, but I think I'll defer that until we have a use case, since I suspect it will need something more complicated than turning off shuffling (providing a list of notebooks to run in order, for instance).